package.json

```diff
@@ -915,6 +927,11 @@
                     "command": "jupyter.notebookeditor.export",
                     "group": "Jupyter",
                     "when": "notebookType == 'jupyter-notebook' && isWorkspaceTrusted && jupyter.ispythonnotebook"
+                },
+                {
+                    "command": "jupyter.replayPylanceLogStep",
+                    "group": "navigation@1",
```

Comment:
Don't we need an icon for this?

Reply:
Nah, it's just for dev. It shows the text. Maybe if it didn't fit?